### PR TITLE
Issue/#1659

### PR DESCRIPTION
**Pull request by jamesrea83**
re https://github.com/adaptlearning/adapt_framework/issues/1659

### CODE REVIEW

File: src/core/js/views/notifyView.js [hunk 1]
```diff
@@ -200,6 +200,8 @@ define([
         },
 
         closeNotify: function (event) {
+            if (!this.model.get('_isActive')) return;
```
Comment (moloko) [inline src/core/js/views/notifyView.js]: Picky I know but `_isOpen` feels like a better name...?

Also, could you add a comment:
```
//prevent from being invoked multiple times - see https://github.com/adaptlearning/adapt_framework/issues/1659
```

File: src/core/js/views/notifyView.js [hunk 2]
```diff
@@ -143,7 +145,7 @@ define([
         },
 
         showNotify: function() {
-
+            this._isOpen = true;
```
Comment (tomgreenfield) [inline src/core/js/views/notifyView.js]: Please remove the underscore from `_isOpen` to match line 17.

File: src/core/js/views/notifyView.js [hunk 3]
```diff
@@ -200,6 +202,9 @@ define([
         },
 
         closeNotify: function (event) {
+            //prevent from being invoked multiple times - see https://github.com/adaptlearning/adapt_framework/issues/1659
+            if (!this._isOpen) return;
+            this._isOpen = false;
```
Comment (tomgreenfield) [inline src/core/js/views/notifyView.js]: Please remove the underscore from `_isOpen` to match line 17.

**Comment by moloko** [comment]
I suppose we should PR this into the 'legacy' branch as well...?

**Comment by oliverfoster** [comment]
Can we retire legacy now?
Put it out to pasture?

**Comment by moloko** [comment]
@oliverfoster we originally said we'd review at the end of 2018. It's not exactly requiring a lot of effort to maintain though, this is actually the first time we've done anything to it since Adapt v3 was released.

**Pull request merged**